.circleci/config.yml

```diff
@@ -273,6 +273,12 @@ workflows:
           target: manifests
           diff-fails: true
           requires: [prepare-env]
+      - go-make:
+          executor: ubuntu
```

Comment:
The other checks above run on the executor `golang`, no? I think they inherit from prepare-env. Does it still fail if you don't specify an executor here?